src/Plugin.Maui.Audio/AudioPlayer/AudioPlayer.android.cs

```diff
@@ -304,6 +304,11 @@ void OnPlaybackEnded(object? sender, EventArgs e)
 	{
 		PlaybackEnded?.Invoke(this, e);
 
+		if (isDisposed)
+		{
+			return;
+		}
+
```

Comment:
Thank you for this. I am wondering if this is the correct fix or whether we should move the `PlaybackEnded?.Invoke(this, e);` line to the end of this method?

I can't decide which is best. What do you think?

Reply:
Arguably the state in which you expect the player to be then `PlaybackEnded` is called is the end of the playback position so I would say it's fine like that. But after checking the other place where `PlaybackEnded` is called seems like it's always the opposite with a Seek(0) before `PlaybackEnded?.Invoke()` so it's probably better to keep it coherent.

I will check if that's work fine and update the PR if you good with this

Reply:
That would be great! Thank you